src/Symfony/Component/Console/Application.php

```diff
@@ -1017,6 +1024,19 @@ private function findAlternatives($name, $collection)
     public function setDefaultCommand($commandName)
     {
         $this->defaultCommand = $commandName;
+
+        return $this;
+    }
+
+    public function setUniqueCommand($commandName)
+    {
+        if (null !== $this->uniqueCommand) {
```

Comment:
Triggering an exception if you configure ``setUniqueCommand('foobar')`` and there is no ``foobar`` command registered